src/components/PeriodDimension/__tests__/PeriodSelector.spec.js

```diff
@@ -1,31 +1,16 @@
-import { shallow } from 'enzyme'
+import { render } from '@testing-library/react'
 import React from 'react'
 import PeriodTransfer from '../PeriodTransfer.js'
 
-describe('The Period Selector component', () => {
-    let props
-    let shallowPeriodTransfer
-
-    const getWrapper = () => {
-        if (!shallowPeriodTransfer) {
-            shallowPeriodTransfer = shallow(<PeriodTransfer {...props} />)
-        }
-        return shallowPeriodTransfer
+test('PeriodSelector matched the snapshot', () => {
+    const props = {
+        initialSelectedPeriods: [],
+        onSelect: jest.fn(),
+        rightFooter: <></>,
+        dataTest: 'period-dimension',
     }
 
-    beforeEach(() => {
-        props = {
-            initialSelectedPeriods: [],
-            onSelect: jest.fn(),
-            rightFooter: <></>,
-            dataTest: 'period-dimension',
-        }
-        shallowPeriodTransfer = undefined
-    })
-
-    it('matches the snapshot', () => {
-        const wrapper = getWrapper()
+    const { container } = render(<PeriodTransfer {...props} />)
 
-        expect(wrapper).toMatchSnapshot()
-    })
+    expect(container).toMatchSnapshot()
```

Comment:
Same problem. Not sure mocking is a great idea to fix this one since the ui is producing a nested component. Maybe just look for the tabs to be in the document.

Reply:
I decided to remove this one since I basically added tests for the `PeriodDimension` component which ultimately just renders `PeriodTransfer`.